cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/AbstractTicket.java

```diff
@@ -119,7 +121,7 @@ public final int getCountOfUses() {
     }
 
     @Override
-    public final long getCreationTime() {
+    public final ZonedDateTime getCreationTime() {
```

Comment:
Don't think this will work with JPA ticket registries. We'll likely run into either schema or serialization issues. Let's keep this as long.